components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -259,6 +233,42 @@ public void deleteBusinessOwner(String businessOwnerId) throws AppManagementExce
     }
 
     /**
+     * Check the business owner usage of web app from the registry.
+     * @param businessOwnerId
+     * @param registry
+     * @param tenantDomain
+     * @return
+     * @throws AppManagementException
+     */
+    public boolean isAssociatedWithApps(String businessOwnerId, Registry registry, String tenantDomain) throws
```

Comment:
Rename this method name to isAssociatedWithApps -> isBusinessOwnerAssociatedWithApps()